distribution/docs/src/main/resources/content/_configuring/remove-default-users-cont.adoc

```diff
@@ -0,0 +1,29 @@
+:title: Removing Default Users Continued
+:type: configuration
+:status: published
+:summary: Remove users.properties file second half.
+:parent: Configuring User Access
+:order: 08
+
+If SSH access to the Karaf shell is to be supported, edit the file `org.apache.karaf.shell.cfg` in the `<INSTALL_HOME>/etc` directory, changing the value
+of the `sshRealm` property from `karaf` to `ldap`.
+
+.Emergency Use of `users.properties` file
+[NOTE]
+====
+Typically, the ${branding} does not manage passwords.
+Authenticators are stored in an external identity management solution. However, ${branding} may be configure the `users.properties` file to include an account with a username and password for emergency use.
```

Comment:
✏️  Can you add a line break at the end of the first sentence?

Is the second sentence supposed to say "${branding} may be configure**d to use** the..." ? or "may configure the `users.properties` file..." ?

Reply:
I didn't write this. I copied and pasted the second half of the `remove-default-users.adoc` to this so I can add another documentation in between. I'm not sure what the second sentence wants to say.